lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -1094,57 +1086,17 @@ describe('AwsInvokeLocal', () => {
         }));
 
     describe('when attempting to build the Java bridge', () => {
-      let awsInvokeLocalMocked;
-      let callJavaBridgeMockedStub;
-
-      beforeEach(() => {
-        mockRequire('child_process', {
-          spawn: () => ({
-            stderr: new EventEmitter().on('data', () => {}),
-            stdout: new EventEmitter().on('data', () => {}),
-            stdin: {
-              write: () => {},
-              end: () => {},
-            },
-            on: (key, callback) => callback(),
-          }),
-        });
-
-        // Remove Node.js internal "require cache" contents and re-require ./index.js
-        delete require.cache[require.resolve('./index')];
-        delete require.cache[require.resolve('child_process')];
-
-        const AwsInvokeLocalMocked = require('./index'); // eslint-disable-line global-require
+      beforeEach(() => {});
```

Comment:
Why those tests were changed? I believe we should only add new tests that concern added improvement

Reply:
There was a `TODO` in there stating that the tests should be refactored so I figured why not:

https://github.com/serverless/serverless/blob/0549d85bc0254a10d3314613892e335da2bc3722/lib/plugins/aws/invokeLocal/index.test.js#L986

Reply:
Thanks @richarddd, that's indeed a good move. Still in that case `beforeEach` statements can be removed entirely (I take there's no point to register noop functions?)